### PR TITLE
fix(docs): fix GitHub repo link

### PR DESCRIPTION
**Pull request by olaf-k**
The current GH link is a button, this replaces it with a `nav-item`.

### CODE REVIEW

File: apps/docs/_layouts/base.njk [hunk 1]
```diff
@@ -83,7 +83,7 @@
           </a>
     </div>
     <dark-mode-toggle	slot="action-items" appearance="toggle"></dark-mode-toggle>
-    <vwc-button slot="action-items" aria-label="View on GitHub" icon="github-mono" icon-trailing href="https://github.com/vonage/vivid-3"></vwc-button>
+    <vwc-nav-item  slot="action-items" icon="github-mono" href="https://github.com/vonage/vivid-3" title="View on GitHub"></vwc-nav-item>
```
Comment (rachelbt) [inline apps/docs/_layouts/base.njk:86]:
```suggestion
    <vwc-nav-item  slot="action-items" icon="github-mono" href="https://github.com/vonage/vivid-3" aria-label="View on GitHub"></vwc-nav-item>
```
the` title `doesn't sink-in to the `a` element.

**Review by rachelbt — COMMENTED** [review]
👍 
@yinonov it seems a little odd that the hamburger icon-button has pill shape but the git-hub link is rounded. 
I guess there's no need for pill shape in menu-item. 
Maybe menu item is not the right fit here at the end :)
Works OK for now, though

**Review by yinonov — APPROVED** [review]
1st PR! 🎊🥳

**Comment by yinonov** [comment]
> 👍 @yinonov it seems a little odd that the hamburger icon-button has pill shape but the git-hub link is rounded. I guess there's no need for pill shape in menu-item. Maybe menu item is not the right fit here at the end :) Works OK for now, though

so no action to take? should change menu shape?

**Comment by olaf-k** [comment]
> > 👍 @yinonov it seems a little odd that the hamburger icon-button has pill shape but the git-hub link is rounded. I guess there's no need for pill shape in menu-item. Maybe menu item is not the right fit here at the end :) Works OK for now, though
> 
> so no action to take? should change menu shape?

I have no strong opinion on this. I can update the PR and remove the pill shape from the menu for consistency's sake.
(Note that the dark/light mode toggle doesn't have any shape btw.)

**Comment by yinonov** [comment]
> > > 👍 @yinonov it seems a little odd that the hamburger icon-button has pill shape but the git-hub link is rounded. I guess there's no need for pill shape in menu-item. Maybe menu item is not the right fit here at the end :) Works OK for now, though
> > 
> > 
> > so no action to take? should change menu shape?
> 
> I have no strong opinion on this. I can update the PR and remove the pill shape from the menu for consistency's sake. (Note that the dark/light mode toggle doesn't have any shape btw.)

can open another PR to resolve the shapes.
regarding the dark mode toggle, it's a google web component and not our own, a tech debt we need to replace soon...

**Comment by yinonov** [comment]
@YonatanKra notice beachball never required a change file for this PR.
Since there are changes to docs site (even though it has no effect on library) we should still publish the site.

**Pull request merged**